From da4c3de2e99f3847b1a1f84c54e3d848ca5f4c8f Mon Sep 17 00:00:00 2001
From: Tony Arcieri <bascule@gmail.com>
Date: Sat, 4 Mar 2023 10:33:39 -0700
Subject: [PATCH] digest+universal-hash: loosen `subtle` version requirement
 (#1260)

Relaxes the version requirement from `=2.4` to `^2.4`, which allows
usage of the newly published `subtle` v2.5.0 release.
---
 digest/Cargo.toml         | 2 +-
 universal-hash/Cargo.toml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/universal-hash/Cargo.toml b/universal-hash/Cargo.toml
index cd954a97..37a41f46 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,7 +41,7 @@
 version = "0.1.6"
 
 [dependencies.subtle]
-version = "=2.4"
+version = "2.4"
 default-features = false
 
 [features]
